internal/web_server/web/views/pages/quiz_pages/username.templ

```diff
@@ -0,0 +1,34 @@
+package quiz_pages
+
+import "github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/layout_components"
+import "github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/icons"
+import "github.com/Molnes/Nyhetsjeger/internal/models/users"
+
+templ UsernamePage(u *users.User) {
+	@layout_components.BaseLayout("Nyhetsjeger - Brukernavn") {
+		<div class="h-screen flex justify-center items-center">
+			<form class="w-1/3 h-3/4 bg-purple-400 flex flex-col items-center rounded-lg" action="" method="post">
+				<h1 class="text-3xl my-4 font-bold">Lag bruker</h1>
+				<div class="w-3/4">
+					<div>
+						<label for="tlfIn" class="inline font-bold mr-4">Mobilnummer</label>
```

Comment:
Use kebab case for IDs. Also consider making them global constants so you can use them in the Handler side. To ensure no discrepancies. See admin_api.go example use for variables from the dashboard_pages package for input names..

Reply:
done